src/statement.cc

```diff
@@ -302,20 +302,22 @@ bool Statement::Bind(const Parameters & parameters) {
 
             switch (field->type) {
                 case SQLITE_INTEGER: {
-                    status = sqlite3_bind_int(_handle, pos,
+                    status = sqlite3_bind_int64(_handle, pos,
                         ((Values::Integer*)field)->value);
                 } break;
                 case SQLITE_FLOAT: {
                     status = sqlite3_bind_double(_handle, pos,
                         ((Values::Float*)field)->value);
                 } break;
                 case SQLITE_TEXT: {
-                    status = sqlite3_bind_text(_handle, pos,
+                    status = sqlite3_bind_text64(_handle, pos,
                         ((Values::Text*)field)->value.c_str(),
-                        ((Values::Text*)field)->value.size(), SQLITE_TRANSIENT);
+                        ((Values::Text*)field)->value.size(),
+                        SQLITE_TRANSIENT,
+                        SQLITE_UTF8);
                 } break;
                 case SQLITE_BLOB: {
-                    status = sqlite3_bind_blob(_handle, pos,
+                    status = sqlite3_bind_blob64(_handle, pos,
```

Comment:
Ref https://github.com/TryGhost/node-sqlite3/blob/master/src/statement.h#L52 for the length field.